JavaScript/ModernSyntax/js-modern-syntax_rest-and-spread/index.js

```diff
@@ -49,3 +54,4 @@ export const result = add(numbers);
 
 const fruits = ["apple", "banana", "orange", "papaya"];
 const anotherFruit = "cherry";
+export const allFruits = {anotherFruit, ...fruits}
```

Comment:
Create a variable `allFruits` which contains `anotherFruit` and `fruits` as a plain array. Here you are creating an object